### DISCUSSION TITLE
Issue when create thumbnail

### DISCUSSION THREAD

**Discussion by ruccvcs50x** [pimcore/pimcore#11599] [Q&A / Support]
Hi there, 

I am trying on fresh install of pimcore - ver 10.3.2 and it show me this issue when create new thumbnail.
How can I fix this issue please?

![image](https://user-images.githubusercontent.com/97378702/156910159-26d4cdde-cf9f-44f3-bf82-ce1ac5591477.png)

**Comment by mrpdean** [comment]
Usually when I've see that sort of error it's often related to incorrect permissions on directories.  

In particular, the web server needs write access to the public/var directory.
